### PR TITLE
[DOCS] Rule execution logs tab renamed to Execution results

### LINKED ISSUE #2259: [DOCS] Rule execution logs tab renamed to Execution results
## Description 

Per https://github.com/elastic/kibana/pull/126063, the “Rule execution logs” tab in rule details has been renamed “Execution results,” along with some cosmetic UI tweaks. There are additional in-progress changes that are currently behind a feature flag for future release, but only the renamed tab and cosmetic changes are GA for 8.4.0.

image

## Tasks

- Update tab name (search/replace wherever possible)
- Update any affected screenshots

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #2259.

Preview: [Monitor and troubleshoot rule executions](https://security-docs_2329.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-monitor.html)

### CODE REVIEW

File: docs/detections/rules-ui-monitor.asciidoc [hunk 1]
```diff
@@ -45,7 +45,7 @@ Use these controls to filter what's included in the logs table:
 * The *Status* drop-down filters the table by rule execution status: 
 ** *Succeeded*: The rule was able to complete its defined search. This doesn't necessarily mean it generated an alert, just that it ran without error.
 ** *Failed*: The rule encountered an error that prevented it from running. For example, a machine learning rule whose corresponding machine learning job wasn't running.
-** *Partial failure*: Nothing prevented the rule from running, but it still might not have returned the expected results. For example, a custom query rule tried to search an index pattern that couldn't be found in {es}.
+** *Warning*: Nothing prevented the rule from running, but it still might not have returned the expected results. For example, a custom query rule tried to search an index pattern that couldn't be found in {es}.
```
Comment (banderror) [inline docs/detections/rules-ui-monitor.asciidoc:48]: Thank you for noticing this one!

**Review by banderror — APPROVED** [review]
LGTM 👍

**Review by benironside — APPROVED** [review]
Looks fresh

**Review by nastasha-solomon — APPROVED** [review]

**Comment by deepikakeshav-qasource** [comment]
Hi @joepeeples,

We have validated the Kibana documentation with reference to this PR and found that the docs are correct

image

image


Hence, we are marking this PR as QA:Approved.

Thank you!

**Review by jmikell821 — APPROVED** [review]
LGTM!

**Pull request merged**